omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
```
$ sudo ls /var/opt/supermarket
data  etc  lib	nginx  postgresql  rails  redis  ssl
```

Looks like we'd want to make `cache_path '/var/opt/supermarket/cache'` just to make sure something cached doesn't clobber actual data directories.

Or—and this might just be silly talk—`/var/cache/supermarket`?